docs/systemd.md

```diff
@@ -25,10 +25,10 @@ Each service has its configuration file located in `/etc/default`. Default
 configurations can be found [here](../docker/package/defaults/).
 
 Files created by the services will be located in `/var/lib/tezos/` by default.
-`tezos-{accuser, baker, endorser}-<protocol>` services can have configurable
+`tezos-{accuser, baker}-<protocol>` services can have configurable
 data directory.
```

Comment:
```suggestion
`tezos-{accuser, baker}-<protocol>` services have a configurable
data directory.
```

or 

```suggestion
`tezos-{accuser, baker}-<protocol>` services' data directories
can be configured.
```

Otherwise it makes ti sound like a on-off setting for allowing configurable data dirs 🤔